selfdrive/car/toyota/carcontroller.py

```diff
@@ -213,13 +213,16 @@ def update(self, sendcan, enabled, CS, frame, actuators,
                                                  ECU.APGS in self.fake_ecus))
     elif ECU.APGS in self.fake_ecus:
       can_sends.append(create_ipas_steer_command(self.packer, 0, 0, True))
-
+    if lead or CS.v_ego < 11.2: # cannot be on below 40 kph
```

Comment:
what is `mini_car`?

Reply:
it seems like you are forced to turn it off below 40kph. Isn't that confusing? What happens if you keep it on at lower speeds?

Reply:
Confusing, yes, but better than always on all the time

Reply:
should 11.2 be replaced by minSetSpeed?

Reply:
That is a very good idea.